pty/ptytest/ptytest.go

```diff
@@ -398,7 +404,13 @@ type PTYCmd struct {
 func (p *PTYCmd) Close() error {
 	p.t.Helper()
 	pErr := p.PTYCmd.Close()
-	eErr := p.outExpecter.close("close")
+	if pErr != nil {
+		p.logf("PTYCmd Close PTYCmd: %v", pErr)
```

Comment:
nit: `PTYCmd.Close() error:`?

`PTYCmd Close PTYCmd` looks weird :)

Reply:
Hehe, true that. It came from the type/method (`(*PTYCmd).Close`) and action (`p.PTYCMD.Close`), so essentially this should've been `PTYCmd Close: PTYCmd Close: %v` which is even worse. Should be better now. 👍🏻